<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SerialPortWid</class>
 <widget class="QWidget" name="SerialPortWid">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>444</width>
    <height>47</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>串口操作</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>:/image/logo.jpg</normaloff>:/image/logo.jpg</iconset>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>串口选择:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QComboBox" name="comBox"/>
   </item>
   <item row="0" column="2">
    <widget class="QComboBox" name="baudComBox">
     <property name="currentText">
      <string>2400</string>
     </property>
     <item>
      <property name="text">
       <string>2400</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>4800</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>9600</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>19200</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>38400</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>57600</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>115200</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="0" column="3">
    <widget class="QPushButton" name="workBtn">
     <property name="text">
      <string>打开串口</string>
     </property>
    </widget>
   </item>
   <item row="0" column="4">
    <widget class="QPushButton" name="updateBtn">
     <property name="text">
      <string>刷新串口</string>
     </property>
    </widget>
   </item>
   <item row="0" column="5">
    <widget class="QLabel" name="stateLab">
     <property name="text">
      <string>----</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
